doc/man/nvme_io_opcode.2

```diff
@@ -56,6 +56,10 @@ enum nvme_io_opcode {
 , 
 .br
 .br
+.BI "    nvme_cmd_cancel"
+,
+.br
+.br
```

Comment:
Please drop the generated doc changes. The docs get generated on release time anyway. Thanks

Reply:
Done, is it ok now?